scripts/gh-projects/examples/mux-video-integration/phase-3/c3.md

```
@@ -10,4 +11,4 @@ Document "How to add a MUX video to a project page" in AGENTS.md.
   - That `<ProjectMuxPlayer>` handles theming automatically via `--project-accent`.
 - No code changes.
 
-Parent: #__PARENT_NUM__
+Parent: #__PARENT_NUM__
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)